myfirst.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+const firebase = require('firebase');
+const { admin, db } = require('./functions/util/admin');
+const config = require('./functions/util/FirebaseConfig');
+const Busboy = require('busboy');
+
+firebase.initializeApp(config);
+var server = express();
+
+server.set('view engine','ejs');
+
+server.get('/',function(req,res){
+  res.render('index');
+});
+
+server.get('/add.ejs',function(req,res){
+  res.render('add');
+});
+server.post('/add.ejs',function(req,res){
+  const Busboy = require('busboy');
+  const path = require('path');
+  const os = require('os');
+  const fs = require('fs');
+
+  const busboy = new Busboy({headers: req.headers});
+  
+  let imageToBeUploaded = {};
+  let imageFileName;
+  
+  busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+      
+      const imageExtension = filename.split('.')[filename.split('.').length - 1];
+      
+      imageFileName = `${Math.round(Math.random() * 1000000000000).toString()}.${imageExtension}`;
+
+      const filepath = path.join(os.tmpdir(), imageFileName);
```

Comment:
Need to check file size being uploaded. Require a threshold for size of picture being updated